src/main/java/dinkplugin/notifiers/CombatTaskNotifier.java

```diff
@@ -117,7 +118,7 @@ private void handle(CombatAchievementTier tier, String task) {
                 .type(NotificationType.COMBAT_ACHIEVEMENT)
                 .text(message)
                 .playerName(player)
-                .extra(new CombatAchievementData(tier, task, taskPoints, totalPoints, tierProgress, tierTotalPoints, completedTier))
+                .extra(new CombatAchievementData(tier, task, taskPoints, totalPoints, tierProgress, tierTotalPoints, GRANDMASTER_TOTAL_POINTS_ID, currentTierName, completedTier))
```

Comment:
Note: `GRANDMASTER_TOTAL_POINTS_ID` is a varbit ID - to get the value you want to do `Client#getVarbitValue` or you could use `cumulativeUnlockPoints.getLastEntry()`

Reply:
I see, so `GRANDMASTER_TOTAL_POINTS_ID = 14814` is just an identifier in the client for the actual value of the GM Points?

Reply:
Correct!